packages/dapp/src/components/buttons/web3Modal.tsx

```diff
@@ -26,9 +26,16 @@ export const SignInButton = () => {
 export const SignOutButton = () => {
   const { isConnecting, signOut } = useAppState();
 
+  const { switchEncryptionAccount } = useAppState();
+  const { switchEncryptionKey } = useAppState();
```

Comment:
the same mentioned in the https://github.com/windingtree/org.id-profile-manager/pull/3/files#r796509285